### PR TITLE
Issue-78 use unix socket again with some docker group tricks

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

File: launch-slave [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -x
```
Comment (rouazana) [inline launch-slave:3]: debug?

File: launch-slave [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+DOCKER_SOCKET=/var/run/docker.sock
+DOCKER_GROUP=docker
+REGULAR_USER=jenkins
+
+if [ -S ${DOCKER_SOCKET} ]; then
+    DOCKER_GID=$(stat -c '%g' ${DOCKER_SOCKET})
+    groupadd -for -g ${DOCKER_GID} ${DOCKER_GROUP}
+    usermod -aG ${DOCKER_GROUP} ${REGULAR_USER}
```
Comment (rouazana) [inline launch-slave:12]: I guess you can remove this line and use --group-add docker instead in the command line

File: workflow-job [hunk 3]
```diff
@@ -213,11 +212,16 @@ def waitForCommandSuccess (command, maxRetries) {
...
 }
 
-env.DOCKER_CERT_PATH = '/keys/'
-env.DOCKER_TLS_VERIFY = 'TRUE'
-
 def setDockerHost(dockerHost) {
     env.DOCKER_HOST = "tcp://${dockerHost}:2376"
+    env.DOCKER_CERT_PATH = '/keys/'
+    env.DOCKER_TLS_VERIFY = 'TRUE'
+}
```
Comment (rouazana) [inline workflow-job:219]: this method can be removed, no?

**Review by aduprat — APPROVED** [review]

File: launch-slave (hunk 1 above)
Comment (mbaechler) [reply launch-slave:3]: yes but I really want to keep it (:

File: launch-slave (hunk 2 above)
Comment (mbaechler) [reply launch-slave:12]: maybe but this one work (:

File: workflow-job (hunk 3 above)
Comment (mbaechler) [reply workflow-job:219]: no, still use for continuous deployment

**Comment by mbaechler** [comment]
we can discuss the PR further if you want, but I'd like to merge to make it to prod

**Pull request merged**